storage/local/persistence.go

```diff
@@ -247,11 +252,29 @@ func newPersistence(
 				Help:      "Quantiles for batch indexing duration in seconds.",
 			},
 		),
-		checkpointDuration: prometheus.NewGauge(prometheus.GaugeOpts{
+		checkpointLastDuration: prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "checkpoint_last_duration_seconds",
+			Help:      "The duration in seconds it took to last checkpoint in-memory metrics and head chunks.",
+		}),
+		checkpointDuration: prometheus.NewSummary(prometheus.SummaryOpts{
 			Namespace: namespace,
 			Subsystem: subsystem,
 			Name:      "checkpoint_duration_seconds",
-			Help:      "The duration in seconds it took to checkpoint in-memory metrics and head chunks.",
+			Help:      "The duration in seconds taken for checkpointing",
```

Comment:
In different news: The decay period is 10m by default, which is only two default checkpoint intervals. It's moot to calculate quantiles for only one or two samples.
Perhaps you intended to not have quantiles? In this case, you need to set `Objectives` explicitly to `map[float64]float64{}`. (Will change in v0.10, but not before because it is a breaking change.)